rpc/glfs-operations.c

```diff
@@ -212,7 +213,11 @@ glusterBlockCreateEntry(struct glfs *glfs, blockCreateCli *blk, char *gbid,
     ret = -1;
     goto out;
   } else {
+#if GFAPI_VERSION760
+    ret = glfs_ftruncate(tgfd, blk->size, NULL, NULL);
```

Comment:
Instead of NULL, it is better to add `struct glfs_stat` prestat and poststat buffers.

Reply:
I checked the source code and it is:

```
2423     if (ret >= 0) {           
2424         if (prestat)          
2425             glfs_iatt_to_statx(glfd->fs, &preiatt, prestat);
2426         if (poststat)         
2427             glfs_iatt_to_statx(glfd->fs, &postiatt, poststat);
2428     }

```
So I just used the NULLs instead currently.

If there is some good reason that we'd better to these buffers, I will update it :-)

Thanks @itisravi .